src/test/java/servise/ActivitiesHandlerTest.java

```diff
@@ -0,0 +1,77 @@
+package servise;
+
+import dto.Activities;
+import dto.Storage;
+import exception.FruitShopException;
+import java.util.Arrays;
+import java.util.List;
+import model.CurrentBalance;
+import model.Fruit;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ActivitiesHandlerTest {
+    private static Fruit bananaFruit = new Fruit("banana");
+
+    @Test
+    public void checkBalanceCase_Ok() {
+        ActivitiesHandler activitiesHandler = new ActivitiesHandler();
+        CurrentBalance currentBalance = new CurrentBalance();
```

Comment:
these fields use in every method, so you can move them to set_up()  @Test method